modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/TeamMembersTable/utils/constants/columns-definitions/RoleColumnType.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ROLE_TYPES} from '../../../../../../../common/utils/constants';
+
+const RoleColumnType = ({roles}) => {
+	const filteredRoles = roles
+		.reduce((rolesAccumulator, role) => {
+			const roleToDisplay = Object.values(ROLE_TYPES).find(
+				(roleType) => roleType.key === role
+			);
+
+			if (roleToDisplay) {
+				rolesAccumulator.push(roleToDisplay.name);
+			}
+
+			return rolesAccumulator;
+		}, [])
+		.join(', ');
```

Comment:
The reducer could do this join internally?

```suggestion
	const filteredRoles = roles
		?.reduce((rolesAccumulator, role) => {
			const roleToDisplay = Object.values(ROLE_TYPES).find(
				(roleType) => roleType.key === role
			);

			if (roleToDisplay) {
				rolesAccumulator.push(roleToDisplay.name);
			}

			return rolesAccumulator;
		}, [])
		.join(', ');
```

Reply:
This logic was removed, check the newest version